internal/inspect/inspect.go

```diff
@@ -42,40 +43,35 @@ func Inspect(out io.Writer, app *types.App, argParameters map[string]string, ima
 		}
 	}, "Service", "Replicas", "Ports", "Image")
 
-	// Add Network section
-	printSection(out, len(config.Networks), func(w io.Writer) {
-		names := make([]string, 0, len(config.Networks))
-		for name := range config.Networks {
-			names = append(names, name)
+	outputSortedStringKeys := func(m interface{}, w io.Writer) {
+		v := reflect.ValueOf(m)
+
+		names := make([]string, 0, v.Len())
+		for _, name := range v.MapKeys() { // panics if m is not a map
+			if name.Kind() != reflect.String {
```

Comment:
I don't think you need to check that, as `outputSortedStringKeys` is a local non-exposed functor, in a totally controlled context.

Reply:
True, I wrote it before I moved it to be a `foo := func(...)` instead of a `func foo(...)` i.e. when it was less localised.

However the reason I kept it is that `reflect.Value.String()`, unlike other similar operators, will accept a non-string here and will try to render them somehow (`<T value>` for the non-string type `T` I think) which seemed like a bit of a bear trap. I suppose it would also be caught be the eventual comparison against `*.golden` in the unit tests though.

Reply:
I dropped this but left a comment about the potentially unexpected behaviour of `String()`.

Reply:
Perfect !